dev/integration_tests/flutter_gallery/test_driver/transitions_perf_e2e_test.dart

```diff
@@ -0,0 +1,19 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// TODO(CareF): avoid the warning message:
+//   VMServiceFlutterDriver: request_data message is taking a long time to complete...
```

Comment:
Do you already have a plan on how to avoid this message?

From here, it's unclear how severe this warning is, and how it might be fixed. If there's no immediate plan to fix this, having more details here or in a Github issue may help a lot in the future.

Reply:
Ahh I should have removed it long before. This is due to the early version of `e2e_driver` doesn't expose timeout API. After the `timeout` is set to a good value this warning is no longer there.